client/src/components/game/gui/GUI.jsx

```diff
@@ -64,6 +65,9 @@ function GUI() {
             PubSub.subscribe(PANEL_CHANGE, () => {
                 setShownPanel(GUIState.activePanel);
             }),
+            PubSub.subscribe(BEFORE_PAGE_UNLOAD, () => {
+                setShownPanel(Panels.CreateAccount);
```

Comment:
We should not show create account if user is already logged in as this might confuse the player.

```js
if (!ApplicationState.loggedIn) {
  setShownPanel(Panels.CreateAccount);
}
```